karate-core/src/test/java/com/intuit/karate/core/KarateHttpMockHandlerTest.java

```diff
@@ -159,7 +159,7 @@ void testKarateRemove() {
         startMockServer();
         run(
                 urlStep(),
-                "path '/hello/1'",
+                "path '/hello', '1'",
```

Comment:
as I expected this would break :| why do you feel strongly about a slash at the start ?

btw we could take a decision to preserve the old behavior by using a `configure pathEncoding = false` or equivalent